### DISCUSSION TITLE
implicit solver with damage case

### DISCUSSION THREAD

**Discussion by yuanzh123** [peridigm/peridigm#312] [Q&A]
Hi,

  I am calculating plate splitting with Peridigm, but the explicit solver needs a tiny time step, leading to a long calculation time for my results. therefore, I changed to the implicit solver. however, it does not converge. do anyone know that an implicit solver can solve damage case? if so, what settings relate to convergency? Is any literature available?

thanks in advance,
Yuan

**Comment by IntePNum** [comment]
I don't know.. Is your question solved?

**Comment by IntePNum** [comment]
Hello, is your question solved???
